### ISSUE TITLE
Error when using the extension with mkdocs 1.1

### ISSUE THREAD

**Issue by Pmeholm** [lukasgeiter/mkdocs-awesome-pages-plugin#19]
Hi, 
First of all thank you for creating this amazing extension. so easy and nice to keep order in a otherwise unstructured documentation base.

Im having a issue when using this extension in mkdocs 1.1. It works however with v1.0.4

```
INFO    -  Building documentation...
Traceback (most recent call last):
  File "C:\Program Files\Python37\lib\runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "C:\Program Files\Python37\lib\runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "C:\Kildekode\Git\Governance\.github\mkdocsBuild\mkserve\lib\site-packages\mkdocs\__main__.py", line 199, in <module>
    cli()
  File "C:\Kildekode\Git\Governance\.github\mkdocsBuild\mkserve\lib\site-packages\click\core.py", line 764, in __call__
    return self.main(*args, **kwargs)
  File "C:\Kildekode\Git\Governance\.github\mkdocsBuild\mkserve\lib\site-packages\click\core.py", line 717, in main
    rv = self.invoke(ctx)
  File "C:\Kildekode\Git\Governance\.github\mkdocsBuild\mkserve\lib\site-packages\click\core.py", line 1137, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "C:\Kildekode\Git\Governance\.github\mkdocsBuild\mkserve\lib\site-packages\click\core.py", line 956, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "C:\Kildekode\Git\Governance\.github\mkdocsBuild\mkserve\lib\site-packages\click\core.py", line 555, in invoke
    return callback(*args, **kwargs)
  File "C:\Kildekode\Git\Governance\.github\mkdocsBuild\mkserve\lib\site-packages\mkdocs\__main__.py", line 143, in serve_command
    **kwargs
  File "C:\Kildekode\Git\Governance\.github\mkdocsBuild\mkserve\lib\site-packages\mkdocs\commands\serve.py", line 141, in serve
    config = builder()
  File "C:\Kildekode\Git\Governance\.github\mkdocsBuild\mkserve\lib\site-packages\mkdocs\commands\serve.py", line 129, in builder
    **kwargs
  File "C:\Kildekode\Git\Governance\.github\mkdocsBuild\mkserve\lib\site-packages\mkdocs\config\base.py", line 197, in load_config
    errors, warnings = cfg.validate()
  File "C:\Kildekode\Git\Governance\.github\mkdocsBuild\mkserve\lib\site-packages\mkdocs\config\base.py", line 107, in validate
    run_failed, run_warnings = self._validate()
  File "C:\Kildekode\Git\Governance\.github\mkdocsBuild\mkserve\lib\site-packages\mkdocs\config\base.py", line 62, in _validate
    self[key] = config_option.validate(value)
  File "C:\Kildekode\Git\Governance\.github\mkdocsBuild\mkserve\lib\site-packages\mkdocs\config\config_options.py", line 130, in validate
    return self.run_validation(value)
  File "C:\Kildekode\Git\Governance\.github\mkdocsBuild\mkserve\lib\site-packages\mkdocs\config\config_options.py", line 591, in run_validation
    plgins[item] = self.load_plugin(item, cfg)
  File "C:\Kildekode\Git\Governance\.github\mkdocsBuild\mkserve\lib\site-packages\mkdocs\config\config_options.py", line 599, in load_plugin
    Plugin = self.installed_plugins[name].load()
  File "C:\Kildekode\Git\Governance\.github\mkdocsBuild\mkserve\lib\site-packages\pkg_resources\__init__.py", line 2443, in load
    return self.resolve()
  File "C:\Kildekode\Git\Governance\.github\mkdocsBuild\mkserve\lib\site-packages\pkg_resources\__init__.py", line 2449, in resolve
    module = __import__(self.module_name, fromlist=['__name__'], level=0)
  File "C:\Kildekode\Git\Governance\.github\mkdocsBuild\mkserve\lib\site-packages\mkdocs_awesome_pages_plugin\plugin.py", line 11, in <module>
    class AwesomePagesPlugin(BasePlugin):
  File "C:\Kildekode\Git\Governance\.github\mkdocsBuild\mkserve\lib\site-packages\mkdocs_awesome_pages_plugin\plugin.py", line 16, in AwesomePagesPlugin
    ('filename', config_options.Type(mkdocs_utils.string_types, default=DEFAULT_META_FILENAME)),
AttributeError: module 'mkdocs.utils' has no attribute 'string_types'
```

the current pip list is 
```
Package                     Version
--------------------------- -------
beautifulsoup4              4.8.2
Click                       7.0
future                      0.18.2
htmlmin                     0.1.12
Jinja2                      2.10.3
jsmin                       2.2.2
livereload                  2.6.1
lunr                        0.5.6
Markdown                    3.2.1
MarkupSafe                  1.1.1
md-mermaid                  0.1.0
mkdocs                      1.1
mkdocs-autolinks-plugin     0.1.0
mkdocs-awesome-pages-plugin 2.2.0
mkdocs-exclude              1.0.2
mkdocs-macros-plugin        0.3.3
mkdocs-material             4.6.3
mkdocs-mermaid-plugin       0.1.1
mkdocs-minify-plugin        0.2.1
nltk                        3.4.5
pep562                      1.0
pip                         20.0.2
Pygments                    2.5.2
pymdown-extensions          7.0b2
PyYAML                      5.3
repackage                   0.7.3
setuptools                  41.2.0
six                         1.14.0
soupsieve                   1.9.5
termcolor                   1.1.0
tornado                     6.0.3
```

**Comment by zoni** [comment]
This should be fixed with `v2.2.1`, so it looks like you're still one minor version behind right now. See also #18.

**Comment by lukasgeiter** [comment]
@zoni is absolutely correct. Updating the plugin to version 2.2.1 should fix the problem.
@Pmeholm I'm closing this issue, if updating does not resolve the error, feel free to re-open it.
